nr-k8s-otel-collector/templates/daemonset-configmap.yaml

```diff
@@ -451,10 +317,15 @@ data:
       batch:
 
     exporters:
-      otlp:
-        endpoint: "nr-k8s-otel-collector-gateway:4317"
-        tls:
-          insecure: true
+      otlphttp:
```

Comment:
Total nit, but could you update this to have a name: `otlphttp/newrelic` so it's apparent that it's outgoing vs just being chained?

Reply:
done